forest/cli/src/cli/wallet_cmd.rs

```diff
@@ -237,3 +254,94 @@ impl WalletCommands {
         }
     }
 }
+
+fn formating_vars(balance_string: String, balance_exact: String) -> (String, String) {
+    if balance_exact.len() <= balance_string.len() {
+        let res = if balance_exact.ends_with(".0") {
+            //unfallible unwrap
+            balance_exact.strip_suffix(".0").unwrap().to_string()
+        } else {
+            balance_exact
+        };
+        (res, "".to_string())
+    } else {
+        //unfallible unwrap
+        let re = Regex::new(r"(\.0+|\.)$").unwrap();
+        let res = re.replace(balance_string.as_str(), "").to_string();
+        (res, "~".to_string())
```

Comment:
I don't think regular expressions are necessary. Try discussing solutions in the slack channel.